src/constants/text.js

```diff
@@ -107,5 +107,12 @@ export default {
   zeroSelected: "0 selected",
   showEvents,
   clearAll: "Clear all",
-  filterEvents: "Filter Events"
+  filterEvents: "Filter Events",
+  noSavedEventsTitle: "Where is the love?",
+  noSavedEventsPart1: "No favourites yet? Tap the",
+  noSavedEventsSaveButtonAlt: "'save this event' button",
+  noSavedEventsPart2: "on any events you like to save them here.",
+  noSavedEventsButton: "Find Events",
+  unSaveEvent: "remove event from saved list",
+  saveEvent: "save this event"
```

Comment:
I think you missed my comment about discoverability of these.

> To make this more discoverable, I suggest starting these with the screen name (`savedEvents...`) for example `savedEventsEmptyTitle`.

I would apply the same to the `unSaveEvent ` and `saveEvent`

Reply:
`unSaveEvent` and `saveEvent` are used on the 💚button on all Event tiles - on the EventDetails, EventListing etc, so they don't really exclusively belong to one screen.

Could change it to `SaveEventButtonSaveEvent` `SaveEventButtonUnSaveEvent`?

Reply:
Yeah, the component name makes sense.